nodejs/lancedb/native.d.ts

```diff
@@ -113,6 +113,7 @@ export class Table {
   countRows(filter?: string | undefined | null): Promise<number>
   delete(predicate: string): Promise<void>
   createIndex(index: Index | undefined | null, column: string, replace?: boolean | undefined | null): Promise<void>
+  update(onlyIf: string | undefined | null, columns: Array<[string, string]>): Promise<void>
```

Comment:
I still prefer `where`, as it is most SQL-y

Reply:
I think it's where in `table.ts`.  In rust, `where` is a reserved word, so you need to do `r#where`.  This is why I used `only_if` in rust.  In the nodejs bindings it appears that napi can't handle `r#where` (it generated `update(r#where: string | undefined | null, ...)` which typescript complained about).  In python I was able to use `where` throughout.  Also, in typescript, `where` is kind of a reserved word.  You can create a function or class/interface property named `where` but you can't create a variable named `where`.

Reply:
Okay I see. That's fine as-is then.

Reply:
I guess we can stick with `where` in rust too.  The user shouldn't need to use `r#where` to call a function named `where` and so they won't be affected.

Reply:
I'll give it a try

Reply:
Looks like that didn't work.  You need to escape `where` even if you are using it as a method name.  E.g. `table.update.r#where("i < 5")`.  That would be pretty annoying so I'm leaving as `only_if`.